public/index.html

```diff
@@ -7,6 +7,39 @@
     <title>2.3-photo-site</title>
 </head>
 <body>
-
+<nav>
+    <div class="left"></div>
+    <div class="right"></div>
+</nav>
+ <div class="header">
+     <img src="" alt="">
+     <p></p>
+ </div>
+ <ul class="posts">
+     <li class="photo">
+         <figure>
+             <img src="" alt="">
+             <figcaption></figcaption>
+             <div class="specs">
+                 <p><i class="fa fa-camera"></i></p>
+                 <p><i class="fa fa-camera"></i></p>
+                 <p><i class="fa fa-camera"></i></p>
+                 <p><i class="fa fa-camera"></i></p>
+                 <p><i class="fa fa-camera"></i></p>
+             </div>
+         </figure>
+     </li>
+     <footer>
```

Comment:
The two photos should link out to what would likely be sold advertising because the web is a terrible place